backend/forecastbox/api/routers/job.py

```diff
@@ -139,13 +139,28 @@ async def update_and_get_progress(job_id: JobId) -> JobProgressResponse:
 
 
 @router.get("/status")
-async def get_status(user = Depends(current_active_user)) -> JobProgressResponses:
-    """Get progress of all tasks recorded in the database."""
+async def get_status(
+    user = Depends(current_active_user),
+    page: int = 1,
+    page_size: int = 10
+) -> JobProgressResponses:
+    """Get progress of all tasks recorded in the database with pagination."""
 
-    job_records = await get_all()
+    if page < 1 or page_size < 1:
+        raise HTTPException(status_code=400, detail="Page and page_size must be greater than 0.")
+
+    job_records = list(await get_all())
```

Comment:
We should probably introduce a stable ordering, I guess on job creation time. I can do so in a future PR if you agree

Reply:
additionally I can add a `limit` to the query etc